build-common/NHibernate.props

```diff
@@ -10,8 +10,10 @@
     <AssemblyVersion>$(VersionMajor).$(VersionMinor).0.0</AssemblyVersion>
     <FileVersion>$(VersionPrefix).0</FileVersion>
 
-    <NhAppTargetFrameworks>net461;netcoreapp2.0</NhAppTargetFrameworks>
-    <NhLibTargetFrameworks>net461;netcoreapp2.0;netstandard2.0</NhLibTargetFrameworks>
+    <NhAppTargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net461;netcoreapp2.0</NhAppTargetFrameworks>
```

Comment:
What is the point of this change? I can debug net461 without problems on my non windows laptop.

Reply:
Really? Every time I try to build Nhiberte fail, says I don’t have .Net Framework 4.6.1

Have you installed Mono on your laptop? I didn’t try to install mono

Reply:
To debug NHibernate on Linux , Should be necessary install mono?